9.1/paper.tex

```diff
@@ -361,8 +361,10 @@ \subsection{Dedicated support for symbolic algebra, including algebra differenti
 and \texttt{SymmetricTensor} classes and associated functions. We have
 also implemented a set of utility functions with the following features:
 \begin{itemize}
-\item \texttt{differentiate} scalar, tensor, and symmetric tensor expressions
-with respect to other scalar, tensor, or symmetric tensor expressions;
+\item \texttt{differentiate} scalar expressions with respect to other scalar expressions, as well as
```

Comment:
You are saying that operations on tensors are different than what you can do with scalars, right? I would probably write this as
```
differentiate scalar, tensor, and symmetric tensors expressions, as well as scalar expressions with respect to other scalar expressions;
```
but it is good enough as is.

Reply:
Different in the sense that the rank of the output object is different from the inputs when doing tensor differentiation. So item 1 should imply that, given scalars `a` and `f(a,T,S)`, with `T` being a tensor and `S` a symmetric tensor, then one can calculate `df/da`, `df/dT` and `df/dS`. Item 2 should similar imply that for a [symmetric] tensor of expressions `F(a,T,S)` we could calculate `dF/da`, `dF/dT` and `dF/dS`. Do you feel this needs to be clarified further, or that I should reformulate this?

Reply:
okay, I guess I still misunderstood what you mean then. It sounds like "with respect to A,B, and C" would be correct as well. Anyways, it doesn't really matter. I was just trying to simplify the language because I had to read the sentence several times trying to figure out what you mean exactly. :-)

Reply:
Well, I'm happy for us to think about this a bit more to come up with a more clear wording. We can leave this PR open until we're all completely satisfied. I'll think about it some more too :-)